### PR TITLE
Add prometheus metrics endpoint

### PR DESCRIPTION
**Pull request by cap10morgan**
ReactiveOps asked if we could expose current active connections as a prometheus metric. Pedestal already publishes an active-servlet-calls metric that this exposes in prometheus format.

### CODE REVIEW

**Comment by cap10morgan** [comment]
BuildKite is having issues

File: src/turbovote_http_api/metrics.clj [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    (fn [ctx]
+      (assoc ctx :response
+                  (prometheus-ring/metrics-response iapetos-registry)))}))
+
+(def set-text-plain-accept-header
+  (interceptor
+   {:name ::add-text-plain-accept-header
+    :enter
+    (fn [ctx]
+      (assoc-in ctx [:request :headers "accept"] "text/plain"))}))
```
Comment (gws) [inline src/turbovote_http_api/metrics.clj]: Is this new interceptor referenced anywhere?

File: src/turbovote_http_api/metrics.clj [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+(def fix-exporter-content-type
+  (interceptor
+   {:name ::fix-exporter-content-type
+    :leave
+    (fn [ctx]
+      (assoc-in ctx [:response :headers "content-type"]
+                TextFormat/CONTENT_TYPE_004))}))
```
Comment (gws) [inline src/turbovote_http_api/metrics.clj]: GitHub is reminding us there's no newline at the end of this file.

**Comment by cap10morgan** [comment]
@gws Still look OK?

**Pull request merged**